### PR TITLE
Add travis

### PR DESCRIPTION
**Pull request by kpcyrd**
This is the travis example config. Let's see if this is working for us.

### CODE REVIEW

**Comment by kpcyrd** [comment]
@benhylau can you help us out on this one?

**Comment by benhylau** [comment]
I am not familiar.
@dpsm any idea?

**Comment by dpsm** [comment]
@benhylau @kpcyrd it seems to not find the project. "The repository at berlinmeshnet/cjdns-android was not found." ?

Does it need to be registered? I don't have permissions so not sure I can help.

**Comment by lgierth** [comment]
Oh yeah, I renamed the organisation from berlinmeshnet to BerlinMeshnet ;)

**Comment by dpsm** [comment]
The question now is how do we re-run the CI job ?

**Comment by benhylau** [comment]
After rebase, build fails with:

```
FAILURE: Build failed with an exception.
* What went wrong:
Some problems were found with the configuration of task ':packageRelease'.
> No value has been specified for property 'signingConfig.storePassword'.
> No value has been specified for property 'signingConfig.keyPassword'.
> No value has been specified for property 'signingConfig.keyAlias'.
```

**Comment by benhylau** [comment]
Success! The question is, what do we want to do with signing? Once we decide on this I will squash all these ugly commits into one before merge, k?

**Comment by kpcyrd** [comment]
I don't believe in signed commits since it's relying on gits crypto system, which is kinda lame-ish. I would prefer not to push broken crypto and don't sign commits.

If you want to you can squash them. Otherwise you can just merge it.

File: build.gradle [hunk 1]
```diff
@@ -30,12 +30,12 @@ android {
         versionCode 1
         versionName "0.0.1-SNAPSHOT"
     }
-    signingConfigs {
-        release
-    }
+//    signingConfigs {
```
Comment (benhylau) [inline build.gradle]: @kpcyrd Oh, not the signed commits. I mean the release signing of the app.

**Comment by benhylau** [comment]
Squashed into https://github.com/BerlinMeshnet/cjdns-android/pull/13

**Pull request closed without merging**